src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -3503,11 +3503,10 @@ public void RunXABuildInParallel ()
 		}
 
 		[Test]
-		public void ProguardBOMError ([Values ("dx")] string dexTool, [Values ("proguard")] string linkTool)
+		public void ProguardBOMError ([Values ("dx", "d8")] string dexTool, [Values ("proguard")] string linkTool)
 		{
 			var proj = new XamarinAndroidApplicationProject () {
 				IsRelease = true,
-				EnableDesugar = true, //It is certain this test would fail without desugar
```

Comment:
Should we just *remove* this property as well?

Reply:
There is still one test that will use it:
https://github.com/xamarin/xamarin-android/blob/f82cd9d696fe9c76b1adc71433a67db9852beb9e/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs#L3542-L3547

This test would fail otherwise, because it has a library with Java 8 features.